### ISSUE TITLE
Unhandled promise rejection Error: Blazor has already started.

### ISSUE THREAD

**Issue by icnocop** [Daddoon/Blazor.Polyfill#67]
Hi.

Thank you for Blazor.Polyfill.

It seems Blazor.Polyfill doesn't explicitly support the scenario where Blazor is configured to start manually.

For example,
```
<script type="text/javascript">
    if (/MSIE \d|Trident.*rv:/.test(navigator.userAgent)) {
        document.write('<script src="@Url.Content("~/lib/blazor.polyfill/blazor.polyfill.min.js")"><\/script>');
    }
</script>
<script src="~/_framework/blazor.server.js" autostart="false"></script>
<script>
        Blazor.start({
            configureSignalR: function (builder) {
                @{
                <environment include="Development">
                    builder.configureLogging("debug");
                </environment>
                }
                builder.withUrl("@Url.Content("~/_blazor")");
            }
        });
</script>
```

Sometimes, this works fine, but other times I get the following error in the browser console:
```
Unhandled promise rejection Error: Blazor has already started.
```

Blazor.Polyfill seems to always force starting on IE:
https://github.com/Daddoon/Blazor.Polyfill/blob/master/src/Daddoon.Blazor.Polyfill.JS/src/Boot.ts#L90

Is this something that has to be explicitly implemented in Blazor.Polyfill, or is there a work-around for this issue?

I'm using Visual Studio 2019, ASP.NET Core 3.1, Blazor Server, blazor.polyfill 3.0.8, Windows 10, and IE11.

Thank you.

**Comment by Daddoon** [comment]
It's a missing feature on Blazor.Polyfill.

However, if the script is forcing an autostart on IE initially, it's only because blazor.server.js rely on some Javascript feature that does not exist on IE, and would prevent Blazor to launch normally.

I don't remember the missing Javascript method name, since it's a long time, but it was something about the script being aware of wich script is actually working and managing code, or something like that. blazor.server.js would in this case fail, and prevent to autostart blazor.

I think that the correct workaround would still be to wait for the Blazor namespace being available in the Blazor.Polyfill code, but instead, if the Blazor object namespace is available, then maybe trying to read the page DOM and see if there is any autostart attribute available on the page, and then respect this attribute value / choice, if available.

**Comment by Daddoon** [comment]
Fixed in 6.0.100.2 !

https://www.nuget.org/packages/BlazorPolyfill.Server/6.0.100.2
https://github.com/Daddoon/Blazor.Polyfill/releases/tag/6.0.100.2

Closing !
